packages/systemd/9012-adjust-default-networkd-dbus-policy.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/src/network/org.freedesktop.network1.conf b/src/network/org.freedesktop.network1.conf
+index 5bd796d69e..11eba43178 100644
+--- a/src/network/org.freedesktop.network1.conf
++++ b/src/network/org.freedesktop.network1.conf
+@@ -20,7 +20,7 @@
+         </policy>
+
+         <policy context="default">
+-                <allow send_destination="org.freedesktop.network1"/>
++                <deny send_destination="org.freedesktop.network1"/>
+                 <allow receive_sender="org.freedesktop.network1"/>
+         </policy>
+
```

Comment:
Why do this?

That's sort of a multi-layered question. Patch files added to the project should always contain a header to work out the reasoning behind the change. There are some historical patches that don't follow that rule, but it's been consistently applied for a while now.

But I'm also not sure why this is necessary - is there a security issue that you're trying to mitigate? If so, how do we know this change is working as intended? What worked before that doesn't work now?

Reply:
Good call out.  I suppose I jumped the gun a bit by including this patch.  I fully expect to need to examine this DBUS policy and further lock it down to avoid unintended or malicious changes to a hosts networking config and stay inline with Bottlerocket's overall security posture.

That being said, I meant for this PR to enable development only and would rather not derail on this topic.  I've removed the patch from this PR and added an item to the [task list](https://github.com/bottlerocket-os/bottlerocket/issues/2449#issuecomment-1334535852) in #2449 to review and implement a proper security policy for this.